resourcepolicies.md

```diff
@@ -304,6 +304,23 @@ Return codes:
 * 403 Forbidden - if you are not logged in with sufficient permissions. See the requirement in the GET Single Resource Policy endpoint
 * 404 Not found - if the resource policy doesn't exist (or was already deleted)
 
+#### Update eperson
+**PUT /api/authz/resourcepolicies/<:id>/eperson**
+
+Update the eperson linked by this resource policy 
+
+Sample CURL command:
+```
+curl -i -X PUT 'https://api7.dspace.org/server/api/authz/resourcepolicies/3/eperson' -H 'Authorization: Bearer eyJhbGciOiJ…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/eperson/epersons/ba05e3dd-aa20-4441-ac05-8ceef6f67ac7'
```

Comment:
I'm not sure why these examples show "dspace7-entities.atmire.com/rest` in the `data`.  These URLs should be updated to point at `api7.dspace.org/server`